CodeChef_problems/Laddu Accural System/problem.md

```diff
@@ -0,0 +1,3 @@
+# Laddu Accural System
+
+Problem: [https://www.codechef.com/laddu](https://www.codechef.com/laddu)
```

Comment:
The problem link seems to be incorrect.
Please correct it 😅

Reply:
I have fixed the link, as well as made the changes that were requested

Reply:
The problem link you have given:https://www.codechef.com/laddu just leads to the Codechef laddu page and not any problem page. Could you please check again

Reply:
Hi, I apologize for the issue. I realized the link leads to the homepage of the competition that had that question and not to the question directly itself. The question can be found at https://www.codechef.com/problems/LADDU , and I have fixed the link for the same in the repo. Again, I would like to apologize for the issue.

Reply:
Oh it's alright 👍